UnityProject/Assets/Scripts/Weapons/Melee/MeleeStun.cs

```diff
@@ -83,7 +83,9 @@ public void ServerPerformInteraction(HandApply interaction)
 			}
 
 			registerPlayerVictim.ServerStun(stunTime);
-			SoundManager.PlayNetworkedAtPos(stunSound, target.transform.position, sourceObj: target.gameObject);
+			// JESTER
+			// SoundManager.PlayNetworkedAtPos(stunSound, target.transform.position, sourceObj: target.gameObject);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)